samples/subsys/usb/audio/headphones_microphone/nrf52840dk_nrf52840.overlay

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
please rename nrf52840dk_nrf52840.overlay to app.overlay, the sample has to work on any system with isochronous endpoints.

Reply:
Done.